docs/reference/commandline/service_create.md

```diff
@@ -0,0 +1,155 @@
+      --stop-grace-period value      Time to wait before force killing a container (default none)
+      --update-delay duration        Delay between updates
+      --update-parallelism uint      Maximum number of tasks updated simultaneously
+  -u, --user string                  Username or UID
+  -w, --workdir string               Working directory inside the container
+```
+
+Creates a service as described by the specified parameters. This command has to
+be run targeting a Manager node.
+
+## Examples
+
+### Create a service
+
+```bash
+$ docker service create --name redis redis:3.0.6
+dmu1ept4cxcfe8k8lhtux3ro3
+
+$ docker service ls
+ID            NAME   REPLICAS  IMAGE        COMMAND
+dmu1ept4cxcf  redis  1/1       redis:3.0.6
+```
+
+### Create a service with 5 tasks
+
+You can set the number of tasks for a service using the `--replicas` option. The
+following command creates a `redis` service with `5` tasks:
+
+```bash
+$ docker service create --name redis --replicas=5 redis:3.0.6
+4cdgfyky7ozwh3htjfw0d12qv
+```
+
+The above command sets the *desired* number of tasks for the service. Even
+though the command returns directly, actual scaling of the service may take
+some time. The `REPLICAS` column shows both the *actual* and *desired* number
+of tasks for the service.
+
+In the following example, the desired number of tasks is set to `5`, but the
+*actual* number is `3`
+
+```bash
+$ docker service ls
+ID            NAME    REPLICAS  IMAGE        COMMAND
+4cdgfyky7ozw  redis   3/5       redis:3.0.7
+```
+
+Once all the tasks are created, the actual number of tasks is equal to the
+desired number:
+
+```bash
+$ docker service ls
+ID            NAME    REPLICAS  IMAGE        COMMAND
+4cdgfyky7ozw  redis   5/5       redis:3.0.7
+```
+
+
+### Create a service with a rolling update constraints
+
+
+```bash
+$ docker service create \
+  --replicas 10 \
+  --name redis \
+  --update-delay 10s \
+  --update-parallelism 2 \
+  redis:3.0.6
+```
+
+When this service is [updated](service_update.md), a rolling update will update
+tasks in batches of `2`, with `10s` between batches.
+
+### Setting environment variables (-e --env)
+
+This sets environmental variables for all tasks in a service. For example:
+
+```bash
+$ docker service create --name redis_2 --replicas 5 --env MYVAR=foo redis:3.0.6
+```
+
+### Set metadata on a service (-l --label)
+
+A label is a `key=value` pair that applies metadata to a service. To label a service with two labels:
+
+```bash
+$ docker service create \
+  --name redis_2 \
+  --label com.example.foo="bar"
+  --label bar=baz \
+  redis:3.0.6
+```
+
+For more information about labels, refer to [apply custom
+metadata](../../userguide/labels-custom-metadata.md)
+
+### Service mode
+
+Is this a replicated service or a global service. A replicated service runs as
+many tasks as specified, while a global service runs on each node in the swarm.
```

Comment:
global service runs on each "active" node in the cluster. (for ex. not on "drain" nodes).